rosetta-source/src/main/rosetta/observable-asset-type.rosetta

```diff
@@ -99,21 +152,38 @@ type Price extends PriceSchedule: <"Specifies a price as a single value to be as
     condition AmountOnlyExists: <"The amount must exist when the price represents a single value, and steps must be absent.">
         value exists and datedValue is absent
 
-type Observable: <"Specifies the object to be observed for a price, it could be an asset or a reference.">
-    [metadata key]
-
-    rateOption FloatingRateOption (0..1) <"Specifies a floating rate index and tenor.">
-        [metadata location]
-    commodity Commodity (0..1) <"Identifies a commodity by referencing a product identifier.">
-        [metadata location]
-    productIdentifier ProductIdentifier (0..*) <"Comprises of an identifier and a source. The associated metadata key denotes the ability to associate a hash value to the ProductIdentifier instantiations for the purpose of model cross-referencing, in support of functionality such as the event effect and the lineage.">
+// type Observable: <"Specifies the object to be observed for a price, it could be an asset or a reference.">
+//     [metadata key]
+
+//     rateOption FloatingRateOption (0..1) <"Specifies a floating rate index and tenor.">
+//         [metadata location]
+//     commodity Commodity (0..1) <"Identifies a commodity by referencing a product identifier.">
+//         [metadata location]
+//     productIdentifier ProductIdentifier (0..*) <"Comprises of an identifier and a source. The associated metadata key denotes the ability to associate a hash value to the ProductIdentifier instantiations for the purpose of model cross-referencing, in support of functionality such as the event effect and the lineage.">
+//         [metadata location]
+//     identifier AssetIdentifier (0..*) <"Comprises of an identifier and a symbology source. The associated metadata key denotes the ability to associate a hash value to the AssetIdentifier instantiations for the purpose of model cross-referencing, in support of functionality such as the event effect and the lineage.  This attribute will replace ProductIdentifier in the full release.">
+//         [metadata location]   
+//     currencyPair QuotedCurrencyPair (0..1) <"Describes the composition of a rate that has been quoted or is to be quoted, including the two currencies and the quotation relationship between the two currencies.">
+//         [metadata location]
+//     optionReferenceType OptionReferenceTypeEnum (0..1) <"The underlying contract which is referenced when determining the final settlement price of the instrument. Eg. Rolling Front Month Future; Spot etc.">
```

Comment:
Don't you need to maintain this `optionReferenceType` attribute in the new `Observable`?

Reply:
I am watching this and hoping not.  I think it is covered by the new ListedDerivative structure as this is not, IMO, an attribute of `Observable`.  The attribute is not referenced anywhere else in the CDM nor DRR

Reply:
Got it. That would be good news. In that case, delete the enum altogether.

Reply:
Done in new PR